src/test/java/org/jabref/logic/importer/fetcher/IacrEprintFetcherTest.java

```diff
@@ -0,0 +1,121 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.util.Optional;
+
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.ImportFormatPreferences;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BiblatexEntryTypes;
+import org.jabref.model.entry.FieldName;
+import org.jabref.testutils.category.FetcherTest;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.mockito.Answers;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+
+@Category(FetcherTest.class)
```

Comment:
When converting to JUnit 5, please replace the category by the `FetcherTest` annotation.